polkadot/runtime/rococo/src/lib.rs

```diff
@@ -199,6 +201,21 @@ parameter_types! {
 
 #[derive_impl(frame_system::config_preludes::RelayChainDefaultConfig)]
 impl frame_system::Config for Runtime {
+	type RuntimeOrigin = RuntimeOrigin;
+	type RuntimeEvent = RuntimeEvent;
+	type RuntimeCall = RuntimeCall;
+	type Hashing = BlakeTwo256;
+	type Lookup = AccountIdLookup<AccountId, ()>;
+	type RuntimeTask = RuntimeTask;
+	type PalletInfo = PalletInfo;
+	type OnNewAccount = ();
+	type OnKilledAccount = ();
+	type OnSetCode = ();
+	type SingleBlockMigrations = ();
+	type MultiBlockMigrator = ();
+	type PreInherents = ();
+	type PostInherents = ();
+	type PostTransactions = ();
```

Comment:
```suggestion
```

Reply:
<img width="1117" alt="Screenshot 2024-05-03 at 19 24 43" src="https://github.com/paritytech/polkadot-sdk/assets/56865496/969d0426-65b8-4cde-a204-e9f8f36fcbe6">

Just like the previous comment, the types should not be needed but I am not sure why the compiler shows this error. Are there any recommendations from your end @bkchr

Reply:
Did you try `cargo build`? `rust-analyzer` is may just confused.

Reply:
It does work, rust-analyzer was truly the problem here. I got it to build without the additional types. Thanks @bkchr